config/config.perfect.yaml

```diff
@@ -0,0 +1,991 @@
+# SPDX-FileCopyrightText: : 2017-2023 The PyPSA-Eur Authors
```

Comment:
I think we agreed on making this containing relative changes only, or? In VScode it is easy to see the relative change with the `Diff: File` command.